package.json

```diff
@@ -82,7 +82,7 @@
     "karma-jasmine": "^0.2.2",
     "karma-jasmine-matchers": "^0.1.3",
     "karma-phantomjs-shim": "^1.0.0",
-    "karma-phantomjs2-launcher": "^0.3.2",
+    "karma-phantomjs-launcher": "^1.0.1",
```

Comment:
It is kind of like a `Gemfile.lock`. It has the exact version of everything installed including nested dependencies. If it is present, `npm install` will use it instead of `package.json` so we use it to make sure we're all installing the same versions of everything.

It's pretty tedious but it ensures we don't have hard to track down bugs due to us using different versions of dependencies. I think Marco has some work related to improving this in #467 though